app-crypt/certbot/certbot-3.2.0-r102.ebuild

```diff
@@ -0,0 +1,256 @@
+	!<app-crypt/certbot-dns-nsone-3.2.0-r100
+	!<app-crypt/certbot-dns-rfc2136-3.2.0-r100
+	!<app-crypt/certbot-nginx-3.2.0-r100
+
+	dev-python/chardet[${PYTHON_USEDEP}]
+	>=dev-python/configargparse-1.5.3[${PYTHON_USEDEP}]
+	>=dev-python/configobj-5.0.6[${PYTHON_USEDEP}]
+	>=dev-python/cryptography-43.0.0[${PYTHON_USEDEP}]
+	>=dev-python/distro-1.0.1[${PYTHON_USEDEP}]
+	>=dev-python/josepy-1.13.0[${PYTHON_USEDEP}]
+	<dev-python/josepy-2[${PYTHON_USEDEP}]
+	>=dev-python/parsedatetime-2.4[${PYTHON_USEDEP}]
+	>=dev-python/pyopenssl-25.0.0[${PYTHON_USEDEP}]
+	dev-python/pyrfc3339[${PYTHON_USEDEP}]
+	>=dev-python/pytz-2019.3[${PYTHON_USEDEP}]
+	>=dev-python/requests-2.20.0[${PYTHON_USEDEP}]
+	certbot-apache? (
+		dev-python/python-augeas[${PYTHON_USEDEP}]
+	)
+	certbot-dns-dnsimple? (
+		>=dev-python/dns-lexicon-3.14.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-dnsmadeeasy? (
+		>=dev-python/dns-lexicon-3.14.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-gehirn? (
+		>=dev-python/dns-lexicon-3.14.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-google? (
+		>=dev-python/google-api-python-client-1.6.5[${PYTHON_USEDEP}]
+		>=dev-python/google-auth-2.16.0[${PYTHON_USEDEP}]
+	)
+	certbot-dns-linode? (
+		>=dev-python/dns-lexicon-3.14.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-luadns? (
+		>=dev-python/dns-lexicon-3.14.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-nsone? (
+		>=dev-python/dns-lexicon-3.14.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-ovh? (
+		>=dev-python/dns-lexicon-3.15.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-rfc2136? (
+		>=dev-python/dnspython-2.6.1[${PYTHON_USEDEP}]
+	)
+	certbot-dns-route53? (
+		>=dev-python/boto3-1.15.15[${PYTHON_USEDEP}]
+	)
+	certbot-dns-sakuracloud? (
+		>=dev-python/dns-lexicon-3.14.1[${PYTHON_USEDEP}]
+	)
+	certbot-nginx? (
+		>=dev-python/pyopenssl-25.0.0[${PYTHON_USEDEP}]
+		>=dev-python/pyparsing-2.4.7[${PYTHON_USEDEP}]
+	)
+	selinux? ( sec-policy/selinux-certbot )
+"
+# RDEPEND+="
+#	!<app-crypt/certbot-dns-cloudflare-3.2.0-r100
+#
+# 	>=dev-python/requests-toolbelt-0.3.0[${PYTHON_USEDEP}] # @TODO is still necessary?
+# 	certbot-dns-cloudflare? (
+# 		# Available in GURU
+# 		>=dev-python/cloudflare-2.19[${PYTHON_USEDEP}]
+# 		<dev-python/cloudflare-2.20[${PYTHON_USEDEP}]
+# 	)
+# 	certbot-dns-digitalocean? (
+# 		# Available in GURU
+# 		>=dev-python/digitalocean-1.11[${PYTHON_USEDEP}]
+# 	)
+# "
+
+# Note: "docs" is not an actual directory under "S", they are actually
+# under each modules, see python_compile_all redefinition, but keep
+# this instruction enabled for dependency configuration.
+distutils_enable_sphinx docs \
+	dev-python/sphinx-rtd-theme
+distutils_enable_tests pytest
+
+# Used to build documentation
+python_compile_all() {
+	use doc || return
+
+	sphinx_compile_all
+
+	# Subdirectory "_build/html" from build_sphinx in eclass/python-utils-r1.eclass
+	mv "${_DISTUTILS_SPHINX_SUBDIR}/_build/html" "${temp_docs}/${dir}" || die
+}
+
+distutils-r1_src_compile() {
+	debug-print-function ${FUNCNAME} "$@"
```

Comment:
That's only for eclasses.

Reply:
Well, as I was reimplementing with a few modification, I thought I should copy what was there.